### PR TITLE
Fix spelling of Organization

### PR DESCRIPTION
**Pull request by JuanitoFatas**
Simple spelling change + emoji submodule bump.

### CODE REVIEW

File: vendor/emojis
Comment (gilesgas) [inline vendor/emojis:1]: Hey @JuanitoFatas !

Thanks very much for picking up on this spelling correction! 🎉 

Had you intended to also commit these emoji changes or did you run `git submodule update` first?
See [Updating vendor\emojis](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#updating-vendoremojis) for more details.

Comment (JuanitoFatas) [reply vendor/emojis:1]: I accidentally included in my commit, so I thought while I am here I can also help with the emoji update and I did run the submodule update git command (hence the force-push). I can remove it from this commit to keep it only a spelling change.

Comment (gilesgas) [reply vendor/emojis:1]: That would be great thanks!

Comment (JuanitoFatas) [reply vendor/emojis:1]: [0c1d572](https://github.com/buildkite/docs/pull/2941/commits/0c1d572485f90b9a7be9ebbc01f02f87c29ce375) :)

**Review by gilesgas — APPROVED** [review]

**Pull request merged**